frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -11,14 +11,8 @@ const enum status {
   BOUND = 'Bound',
   AVAILABLE = 'Available',
 }
-export const filterCephAlerts = (alerts: Alert[]): Alert[] => {
-  const rookRegex = /.*rook.*/;
-  return alerts?.filter(
-    (alert) =>
-      alert?.annotations?.storage_type === 'ceph' ||
-      Object.values(alert?.labels)?.some((item) => rookRegex.test(item)),
-  );
-};
+export const filterCephAlerts = (alerts: Alert[]): Alert[] =>
+  alerts.filter((alert) => _.get(alert, 'annotations.storage_type') === 'ceph');
```

Comment:
The inclusion of Ceph alerts in the Storage Overview might be one thing from the original PR that could stay?

Reply:
Yes let's keep this part as it is.